server/events/project_command_builder_internal_test.go

```diff
@@ -199,6 +202,7 @@ workflows:
 			repoCfg: `
 version: 3
 automerge: true
+locking: false
```

Comment:
I'm not in love with this keyword. My understanding is there's still locking, just not during planning. This would lead a naive user (me) to believe there are no locks during apply either. 

I don't have a better idea though

Reply:
maybe change the name to `pr_locks`?

Reply:
or `plan_locks`